api/pom.xml

```diff
@@ -26,6 +26,10 @@
       <artifactId>jakarta.enterprise.cdi-api</artifactId>
       <scope>provided</scope>
     </dependency>
+    <dependency>
```

Comment:
@geoand you'll be interested in this (probably not in a positive way), we're introducing a JSON-P dependency from the API module...

Reply:
@jmartisk we are already pulling the implementation in Quarkus - https://github.com/quarkusio/quarkus/blob/main/extensions/smallrye-health/runtime/pom.xml#L17-L25. Which is bringing this - https://github.com/smallrye/smallrye-health/blob/main/implementation/pom.xml#L67-L70. 

Not sure what you mean could be a problem?

Reply:
That it leaks JSON-P into our public API

Reply:
I'm not saying we shouldn't do that, but there have been discussions that we should move everything from JSON-P to Jackson, and this will make it harder, hence I'm letting @geoand know

Reply:
Thanks a lot for the heads up @jmartisk. Just so I am up to speed, currently SR Health does not expose any Json-P types in the API surface, correct?

Reply:
That shouldn't be an issue since we already have this https://github.com/smallrye/smallrye-health/blob/main/implementation/src/main/java/io/smallrye/health/SmallRyeHealth.java#L17

Reply:
But note that I'm thinking whether I can rewrite SR Health to use Jackson but that will be breaking change.

Reply:
AFAIK we're thinking of introducing an abstraction over JSON-B, JSON-P and Jackson. Ok, since we already leak it somewhere else, this change is not that much of a problem :)

Reply:
Correct on both counts